pkg/gadgets/run/tracer/tracer.go

```diff
@@ -376,6 +376,22 @@ func (t *Tracer) attachProgram(gadgetCtx gadgets.GadgetContext, p *ebpf.ProgramS
 		case strings.HasPrefix(p.SectionName, "kretprobe/"):
 			logger.Debugf("Attaching kretprobe %q to %q", p.Name, p.AttachTo)
 			return link.Kretprobe(p.AttachTo, prog, nil)
+		case strings.HasPrefix(p.SectionName, "uprobe/"):
+			logger.Debugf("Attaching uprobe %q to %q", p.Name, p.AttachTo)
+			parts := strings.Split(p.AttachTo, ":")
+			ex, err := link.OpenExecutable(parts[0])
```

Comment:
To open executables from all containers, you can look at the following packages for inspiration:
* Socket filter ebpf programs:
  * https://github.com/inspektor-gadget/inspektor-gadget/blob/main/docs/devel/network-gadget-dispatcher.png
  * https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/networktracer/tracer.go#L15-L31
* tc ebpf programs:
  * https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/tchandler/tracer.go#L15-L19